### ISSUE TITLE
Password first character only

### ISSUE THREAD

**Issue by mattheyan** [kirankumarcelestial/WindowsCredientialManager#3]
Cool project! Unfortunately I'm running into problems.

After finally getting it to build (node-gyp vcpp issues) I was able to run the sample successfully. However, when I read a pre-existing password it only includes the first character. As an example, I am able to read credentials successfully with [this tool](http://www.powershellgallery.com/packages/CredentialManager/1.0), whether created with the tool or through the Credential Manager UI. ...but, the node module only returns the first character of the password. If I use the powershell module to read credentials that were created by the node module, they print out as squares, so maybe an encoding issue?

In summary, a simple test on my end was to create credentials through the Credential Manager UI, and then attempt to read them with the node module.

**Comment by kirankumarcelestial** [comment]
Cred Manager 1.0 was rewritten to support Node v12 Bindings. Will fix it in next couple of days.
